firefox-ios/firefox-ios-tests/Tests/SyncIntegrationTests/xcodebuild.py

```diff
@@ -57,7 +61,7 @@ def test(self, identifier, erase=True):
                 cwd=os.chdir("../../.."),
                 stderr=subprocess.STDOUT,
                 universal_newlines=True)
-        except subprocess.CalledProcessError as e:
+        except Exception as e:
```

Comment:
What was the reason to just catch a generic `Exception` here instead of the more specific `CalledProcessError`?

Reply:
Oh I don't quite remember. I can change this back if you'd like